README.md

```diff
@@ -74,7 +74,7 @@ The options object is used to generate the `Set-Cookie` header of the session co
 * `domain` - The `Domain` attribute.
 
 ##### store
-A session store. Needs the following methods: 
+A session store. The store should be an event emitter with the following methods: 
```

Comment:
I don't get the change. It's not an event emitter.

Reply:
Actually [it's an event emitter](https://github.com/fastify/session/blob/4546ff96cd3de71a4f33e6b11522adc5f8a3ad6b/lib/store.js#L8), and there is a [test](https://github.com/fastify/session/blob/4546ff96cd3de71a4f33e6b11522adc5f8a3ad6b/test/store.test.js#L109-L116) for this.

While I do not fully understand why it needs to be an event emitter (since the event emitter API is not used anywhere), I believe that this originated from `express-session` to [track if the store is disconnected](https://github.com/expressjs/session/blob/0048bcac451ad867299d404aca94c79cc8bc751d/index.js#L188), and I guess this library aiming for compatibility with `express-session` stores ported it, but did not implemented store connection tracking.

Do you think I should remove it?

Reply:
Meanwhile this is due to the `express-session` compatibility it's not required to be an event emitter. I feel it's just a bootstrap for the direct compatibility. Only the described methods are required.
I feel it might be misinterpreted by end users. Might be worth adding a comment in the code for this (in another PR)

Reply:
> might be misinterpreted by end users

You're right. I'll remove it.